### ISSUE TITLE
Governance process for data sovereignty checks of credentials and policy purposes

### ISSUE THREAD

**Issue by DanielaWuensch** [eclipse-tractusx/sig-release#583]
### Situation
As of today (Catena-X Release 24.05), we have a set of credentials and a set of policy purposes defined in the CX ODRL profile. We know already that changes will come such as
- new verifiable credentials will be introduces
- new purposes will be set to deprecated and finally removed
- existing purposes will be set to deprecated and finally removed
- existing constraints will be set to mandatory or to optional
### Tasks to be done for this ticket
To define what happens in such cases, governance rules need to be defined what happens in these cases
- e.g. both are valid in parallel 
- e.g. context what version is valid clarified upfront

## Impact = Outcome
Governance rules should be defined in 
- CX kits
- new versions of purposes and verifiable credentials

CX standards as part of Houston process will be linked

## Additional information 

- [x] I'm willing to contribute to this feature

**Comment by DanielaWuensch** [comment]
Labels: "SSI" and "data sovereignty" will be added as soon as they exist

**Comment by DanielaWuensch** [comment]
In refinement we might decide that it is not relevant for edc and no change necessary there.

**Comment by DanielaWuensch** [comment]
@jjeroch : Thanks for the feedback. I structured it differently to make clear that it is only about defining the process.

**Comment by DanielaWuensch** [comment]
@DanielaWuensch : As follow up, Houston planning will be linked.

**Comment by HFocken** [comment]
I'd like to contribute

**Comment by stephanbcbauer** [comment]
Hello @DanielaWuensch , @HFocken 

Since the feature is a 24.08 feature and the development phase is coming to an end, we need a status on the feature.

- [ ] Currently you are assigned (Responsible) → Is this correct? If not, please assign the correct contact person
- [ ] Please check whether the status (backlog, work in progress ...) is set correctly
- [ ] Please check whether there is an assignment to a committee or expert group (supported-by) to be found on the [board](https://github.com/orgs/eclipse-tractusx/projects/26/views/28)
- [ ] Please comment on the current status of the feature
- [ ] Are all SubTasks (issues from other repositories that deal with the feature) linked? → The easiest way is to mention the feature here in the issue (via the ID) so we can see which teams/repositories are involved.

If you need any clarification, please get in touch, thank you very much.
Stephan

**Comment by stephanbcbauer** [comment]
Hello @DanielaWuensch 
 
In a few weeks, we want to publish Tractus-X Release 24.08. For this, we need a clear status about the features (which will be included in the release and which will be cancelled or postponed to the next release).
 
Currently, the feature is still in status `Work in progress`. Please update the status by 09.07.24 eob.
 
If the feature is not ready for release 24.08, or it is possibly not planned at all, please set the status to `Not in release`. Please remember that in this case, any dependent components must be informed.
 
Thank you and best regards.

**Comment by stephanbcbauer** [comment]
@DanielaWuensch, @HFocken is on a leave... did you receive some information from him?

**Comment by DanielaWuensch** [comment]
Feature was not realized in 24.08 as we focused on initial introduction of framework agreement and stabilization for release 24.08. Thus, we move it 24.12.

**Comment by stephanbcbauer** [comment]
> was presented in open planning ⇾ no implementation in tractus-x needed

Please be aware no implementation in tractus-x needed

**Comment by DanielaWuensch** [comment]
@arnoweiss : Could your take over review as committer?

**Comment by matbmoser** [comment]
@stephanbcbauer  R24.08 needs to be updated to R24.12, shall I just change it or I need to add something?

**Comment by jSchuetz88** [comment]
> [@stephanbcbauer](https://github.com/stephanbcbauer) R24.08 needs to be updated to R24.12, shall I just change it or I need to add something?
@stephanbcbauer said, update the milestone when there are no concerns with and when there are committers.

**Comment by stephanbcbauer** [comment]
> > [@stephanbcbauer](https://github.com/stephanbcbauer) R24.08 needs to be updated to R24.12, shall I just change it or I need to add something?
> > [@stephanbcbauer](https://github.com/stephanbcbauer) said, update the milestone when there are no concerns with and when there are committers.

Depends on the dependencies :) Maybe we can discuss this in the technical status sync on thursday?

**Comment by guenterban** [comment]
Closed by NSC because no progress. Please reopen and assign a propper release if needed.
